File: TaskManager/src/IssueRepository.java
import java.util.HashMap;
class IssueRepository {
    private static IssueRepository rep;
    private final HashMap<String, Issue> issues = new HashMap<String, Issue>();

    IssueRepository() {}

    public static IssueRepository getInstance() {
        if (rep == null) {
            rep = new IssueRepository();
        }
        return rep;
    }

    public void createIssue(Issue issue) {
        issues.put(issue.getId(), issue);
    }

    public Issue getIssue(String id) {
        return issues.get(id);
    }

    public void removeIssue(String id) {
        issues.remove(id);
    }

    public void showAllIssues() {
        issues.values().forEach(System.out::println);
    }

    public HashMap<String, Issue> getList() {
        return issues;
    }
}

File: TaskManager/IssueRepository.cs
using System;

public class IssueRepository
{
    private Dictionary<int, Issue> issues;

    public void AddIssue(Issue issue)
    {
        issues.Add(issue.ID, issue);
    }
    public void GetIssue(int ID)
    {
        return issues[ID];
    }
    public void UpdateIssue(Issue issue)
    {
        issues[issue.ID] = issue;
    }
}
